update/files/multiFileReader.go

```diff
@@ -100,11 +100,13 @@ func (m *multiFileReader) ReadNextItem(fileName string) (string, []byte, error)
 	}
 
 	hexEncoded := scanner.Text()
+
 	key, err := hex.DecodeString(hexEncoded)
 	if err != nil {
 		return "", nil, err
 	}
 
+	log.Trace("import key", "key", string(key))
```

Comment:
"import" is enough as "key" is printed after

Reply:
done